services/fuel-core/Dockerfile

```diff
@@ -21,7 +22,6 @@ CMD exec ./fuel-core run \
     --ip ${IP} \
     --port ${PORT} \
     --db-path ${DB_PATH} \
-    --utxo-validation \
     --min-gas-price ${MIN_GAS_PRICE} \
```

Comment:
I think if it works without could be an issue once we have some predicate tests that would require this to be enabled. @Voxelot do you have some idea why this could be happening?